api/src/pcapi/alembic/versions/20220525T150715_f96e62709e85_create_cinema_provider_pivot_table.py

```diff
@@ -0,0 +1,45 @@
+"""create_cinema_provider_pivot_table
```

Comment:
```suggestion
""" Create cinema_provider_pivot table
```
Ce texte est un message que l'on peut rédiger en langage plus humain, et qui s'affiche par exemple dans la console lorsque l'on joue la migration.